src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditContainer.tsx

```diff
@@ -105,15 +99,32 @@ const WorkPackageEditContainer: React.FC<WorkPackageEditContainerProps> = ({ wor
     append: appendDeliverable,
     remove: removeDeliverable
   } = useFieldArray({ control, name: 'deliverables' });
-  const { fields: blockedBy, append: appendBlocker, remove: removeBlocker } = useFieldArray({ control, name: 'blockedBy' });
+
+  const {
+    data: project,
+    isLoading: projectIsLoading,
+    isError: projectIsError,
+    error: projectError
+  } = useSingleProject(validateWBS(projectWbsPipe(workPackage.wbsNum)));
 
   const { mutateAsync } = useEditWorkPackage(workPackage.wbsNum);
 
-  if (allUsers.isLoading || !allUsers.data || !auth.user) return <LoadingIndicator />;
+  if (allUsers.isLoading || !allUsers.data || projectIsLoading || !project) return <LoadingIndicator />;
   if (allUsers.isError) {
     return <ErrorPage message={allUsers.error?.message} />;
   }
-  const { userId } = auth.user;
+  if (projectIsError) {
+    return <ErrorPage message={projectError?.message} />;
+  }
+
+  const blockedByToAutocompleteOption = (workPackage: WorkPackage) => {
+    return { id: wbsPipe(workPackage.wbsNum), label: `${wbsPipe(workPackage.wbsNum)} - ${workPackage.name}` };
+  };
+
+  const blockedByOptions =
+    project.workPackages.filter((wp) => wp.id !== workPackage.id).map(blockedByToAutocompleteOption) || [];
```

Comment:
wps can only be blocked by wps in the same project?

Reply:
yeah lol